src/FunctionApps/python/GenerateCSVs/patient.py

```diff
@@ -0,0 +1,166 @@
+PATIENT_COLUMNS = [
+    "patient_hash",
+    "given_name",
+    "family_name",
+    "birthDate",
+    "gender",
+    "street",
+    "city",
+    "state",
+    "postalCode",
+    "latitude",
+    "longitude",
+    "race",
+    "ethnicity",
+]
+
+
+def parse_patient_resource(pt_rsc: dict) -> str:
```

Comment:
Awesome, but it's probably easier to work with downstream if you return `List[Any]` so other functions can jam their `List`s on the end